tiatoolbox/wsicore/wsireader.py

```diff
@@ -412,9 +564,9 @@ def read_rect(
             :class:`numpy.ndarray`: array of size MxNx3 M=size[0], N=size[1]
 
         Example:
-            >>> from tiatoolbox.wsicore.wsireader import get_wsireader
+            >>> from tiatoolbox.wsicore import wsireader
```

Comment:
please restore get_wsireader

Reply:
Fixed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/123/commits/3fbfef2da40097681f72adf4f0d16bd7377f42e1